deploy/kubernetes/webhook-example/deployment.yaml

```diff
@@ -17,7 +17,8 @@ spec:
     spec:
       containers:
       - name: filestorecsi-validation
-        image: gcr.io/saikatroyc-test/gcp-filestore-csi-driver-webhook:v2
+        # change the following image to a correct image destimation
```

Comment:
change the following to the correct image url

Reply:
done